plugins/tokenVoting/pages/proposal.tsx

```diff
@@ -83,8 +97,23 @@ export default function ProposalDetail({ id: proposalId }: { id: string }) {
   };
 
   useEffect(() => {
-    if (voteResponse) addAlert("Your vote has been submitted", voteResponse);
-  }, [voteResponse]);
+    if (status === "idle" || status === "pending") return;
+    else if (status === "error") {
+      if (error?.message?.startsWith("User rejected the request")) return;
+      alert("Could not create the proposal");
+      return;
+    }
+
+    // success
+    if (!votingTxHash) return;
+    else if (isConfirming) {
+      addAlert("The vote has been submitted", votingTxHash);
+      return;
+    } else if (!isConfirmed) return;
+
+    // addAlert("The vote has been registered", votingTxHash);
+    reload();
```

Comment:
We should not be reloading. If something isn't refreshing by itself we fix that in particular. But refreshing the whole page for that isn't the way!